stdlib/2and3/curses/__init__.pyi

```diff
@@ -12,4 +12,4 @@ COLS: int
 COLORS: int
 COLOR_PAIRS: int
 
-def wrapper(func: Callable[..., _T], *arg: Any, **kwds: Any) -> _T: ...
+def wrapper(__func: Callable[..., _T], *arg: Any, **kwds: Any) -> _T: ...
```

Comment:
I don't think this is positional-only?

```
In [27]: curses.wrapper??
Signature: curses.wrapper(func, *args, **kwds)
Source:   
def wrapper(func, *args, **kwds):
    """Wrapper function that initializes curses and calls another function,
```
I'm on 3.7

Reply:
The text signature has it as positional-only, https://github.com/python/cpython/blob/3.8/Lib/curses/__init__.py#L113
(It's also positional-only in 3.9, for whatever that's worth)

Reply:
I see, looks like they deprecated passing it as a keyword in 3.8 and removed it in 3.9. I suppose strictly we should mark it as positional-only in 3.9+ only, but it should be fine to just have it unconditionally positional-only, at least as long as nobody actually complains.